src/masternodes/rpc_vault.cpp

```diff
@@ -433,10 +434,13 @@ UniValue listvaults(const JSONRPCRequest& request) {
     std::string loanSchemeId;
     VaultState state{VaultState::Unknown};
     bool verbose{false};
+    bool isMine{false};
     if (request.params.size() > 0) {
         UniValue optionsObj = request.params[0].get_obj();
         if (!optionsObj["ownerAddress"].isNull()) {
-            ownerAddress = DecodeScript(optionsObj["ownerAddress"].getValStr());
+            isMine = (optionsObj["ownerAddress"].getValStr() == "mine");
```

Comment:
Could you lift `getValStr()` into a variable. This is not a hotspot, but a good practise to start lifting these up.